lib/rubycritic/analysers/smells/reek.rb

```diff
@@ -26,6 +26,8 @@ def to_s
 
       private
 
+      attr_reader :analysed_modules
+
       def add_smells_to(analysed_module)
         Reek.new(analysed_module.pathname).smells.each do |smell|
```

Comment:
Maybe by using `Reek::Examiner` instead of the top level module will work?

`examiner = Reek::Examiner.new(source)`

Also, this `examiner` can be memoized.

Reply:
This is not using the top level module, he is using the `lib/rubycritic/analysers/helpers/reek.rb` module, I tried use the examiner this way, passing the configuration and still dont work

Doing some research on reek's source code I see the `excluded_paths` param is filtered before he reach the `Reek::Examiner`, in [SourceLocator](https://github.com/troessner/reek/blob/master/lib/reek/source/source_locator.rb).